### ISSUE TITLE
Resistance to thyroid hormone

### ISSUE THREAD

**Issue by euniceyi** [obophenotype/human-phenotype-ontology#4733]
This is a request to add an agnostic/NOS term for resistance to thyroid hormone given that the currently most applicable term of **Thyroid hormone receptor defect** implies a specific etiology that is not always known in observed RTH phenotypes. Thus we recommend making **Thyroid hormone receptor defect** a child of the new term **Resistance to thyroid hormone** which in turn will be a child of **Abnormality of thyroid physiology** to accommodate phenotypes with serum levels implying TH resistance but without known mechanism.

**Preferred term label:**
Resistance to thyroid hormone

**Synonyms**
Thyroid hormone action defect

**Definition**
"Elevated serum levels of free thyroid hormone with nonsuppressed TSH."
PMID: 24847459 

**Parent term**
Abnormality of thyroid physiology
https://hpo.jax.org/app/browse/term/HP:0002926

**Child term**
Thyroid hormone receptor defect
https://hpo.jax.org/app/browse/term/HP:0002930

**Diseases characterized by this term?**
Generalized Resistance To Thyroid Hormone
ORPHA:3221

**Your nano-attribution (ORCID)**
0000-0001-6328-7035

**Comment by nicolevasilevsky** [comment]
@pnrobinson is it okay to add this term? I can do so

**Comment by pnrobinson** [comment]
@euniceyi -- sorry for the delay. The HPO does not intend to code genetic findings, and so I think that the current term, "Thyroid hormone receptor defect" really intends to mean 
"Impaired sensitivity to thyroid hormone". We would not want to have a term that describes "Impaired sensitivity to thyroid hormone due to a mutation in THRA" ( or "....THRB"). 
The current term is a little inconsistent because it has as exact synonym
1. End-organ unresponsiveness to thyroid hormone
2. Resistance to thyroid hormone 

and I think looking back it was a mistake to choose the primary label to be "thyroid hormone receptor defect"

So I would propose the label
"Impaired sensitivity to thyroid hormone"
Def: Reduced sensitivity of end organs to thyroid hormone characterized by elevated serum levels of free thyroid hormone with nonsuppressed thyroid stimulating hormone.
Comment: Impaired sensitivity to thyroid hormone may be accompanied by goiter and no clear symptoms and signs of thyrotoxicosis. The etiologies of this feature include defects in the thyroid hormone receptor, reduced cellular access of the biologically active thyroid hormone, T3 related to defects of thyroid hormone cell membrane transport, and a defect reducing the intracellular metabolism generating T3 from T4.
PMID: 24847459


Is this acceptable?
Elevated serum levels of free thyroid hormone with nonsuppressed TSH

**Comment by euniceyi** [comment]
@pnrobinson Thank you very much for your thoughtful response! This is all eminently reasonable and changing the label to "impaired sensitivity to thyroid hormone" would be very helpful for our current project as well. 

Will this new label replace "thyroid hormone receptor defect" completely--thus eliminating any mechanistic or genetic description--or do you think it still may be useful in some way to keep "thyroid hormone receptor defect" but as a subclass of the new term ("impaired sensitivity to thyroid hormone")?

**Comment by pnrobinson** [comment]
The HPO does not encode etiology or genetics -- it would reduce the utility of the ontology if we did, since it would encode our current knowledge and impede new discoveries. So I do not think it would be good to have subclasses for the etiologies. We should ensure that we are annotating the diseases accordingly.

**Comment by pnrobinson** [comment]
implemented this. It should be visible with the next HPO release in 1 month.
